clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -365,8 +397,16 @@ void SYCL::gen::BackendCompiler::ConstructJob(Compilation &C,
           getToolChain().getTriple().getArch() == llvm::Triple::spir64) &&
          "Unsupported target");
   ArgStringList CmdArgs{"-output",  Output.getFilename()};
+  ArgStringList ForeachArgs;
   for (const auto &II : Inputs) {
     CmdArgs.push_back("-file");
+    if (II.getType() == types::TY_Tempfilelist) {
+      std::string FileName(II.getFilename());
```

Comment:
Suggest initializing before entering the condition and reusing [2]

Reply:
Okay.